contracts/FlowTransactionSchedulerUtils.cdc

```diff
@@ -320,9 +322,14 @@ access(all) contract FlowTransactionSchedulerUtils {
                 let ids = self.idsByTimestamp[timestamp] ?? []
                 for id in ids {
                     let status = FlowTransactionScheduler.getStatus(id: id)
-                    if status == nil || status == FlowTransactionScheduler.Status.Unknown {
-                        transactionsToRemove.append(id)
-                    }
+                    transactionsToRemove.append(id)
+                }
```

Comment:
by removing the conditional, I think we're appending adding ids that could be scheduled. is that right?

Reply:
It wouldn't remove scheduled ids because it is only doing timestamps in the past, but now that you mention it, I think it might remove ids that are supposed to execute in the same block but haven't actually been executed yet. I need to think about it a bit more, but I don't think that is an issue because this just removes the scheduled transaction resource and does not affect the handler

Reply:
I'll still add a check that the status can't be scheduled though just to be safe